component.js

```diff
@@ -274,7 +286,7 @@ function factory (initialOptions) {
       }
 
       return create;
-    };
+    }
```

Comment:
Semis are for expressions! This is a declaration https://github.com/omniscientjs/omniscient/blob/4d53f7c2c905e99901c90e82e042d588481d7682/component.js#L175